testing/command_helper.go

```diff
@@ -470,6 +473,96 @@ func (k *Kubectl) WaitForData(namespace string, resourceName string, name string
 	})
 }
 
+func writeTemporaryYaml(v interface{}) (string, error) {
```

Comment:
nit: YAML - abbreviations are capitalized in golang

Reply:
Thanks! fixed :+1: